tket2/src/static_circ.rs

```diff
@@ -0,0 +1,265 @@
+//! A 2d array-like representation of simple quantum circuits.
+
+mod match_op;
+
+use std::{collections::BTreeMap, fmt, rc::Rc};
+
+use hugr::{Direction, HugrView, Port, PortIndex};
+pub(crate) use match_op::MatchOp;
+
+use derive_more::{From, Into};
+use thiserror::Error;
+
+use crate::{circuit::units::filter, Circuit};
+
+/// A circuit with a fixed number of qubits numbered from 0 to `num_qubits - 1`.
+#[derive(Clone, Default)]
+pub struct StaticSizeCircuit {
+    /// All quantum operations on qubits.
```

Comment:
Clarify a bit what's each vector about.
```suggestion
    /// All quantum operations on each qubit.
```